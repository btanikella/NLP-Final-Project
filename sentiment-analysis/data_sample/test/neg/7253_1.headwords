afraid
is
reminiscent
From ...
say
enough
one
