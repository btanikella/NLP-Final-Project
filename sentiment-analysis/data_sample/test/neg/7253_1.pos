CC PRP VBP JJ IN PRP MD RB VB WRB
PRP RB VBZ DT RB JJ CC RB JJ NN
IN DT NNS PRP VBZ JJ IN JJ NN NN WRB DT NN VBD VBN VBN RP IN DT NNP RB RB IN IN WDT VBD VBG IN
IN DT JJ CC JJ NN -LRB- NNS VBP TO VB IN CD NNS RB -RRB- RB VBN IN NN  TO DT NNS WP VBP RB RB VBN IN NN IN  JJ  NNS VBD RB IN NN  TO DT RB JJ NN IN DT JJ NN  TO DT NN IN DT NN JJ WRB NN NN NNS VBP DT JJ NNS RB IN JJ NN  CC RB DT JJ NN WDT VBZ IN DT NN NN JJR IN DT NN : PRP
RB MD RB VB WRB JJ DT VBZ
RB DT NN TO VB IN DT JJ NNP NNP VBD VBD RB RB TO VB PRP VBG IN JJR IN DT NN
DT RB VBZ CD TO VB VBN IN NN NNS
