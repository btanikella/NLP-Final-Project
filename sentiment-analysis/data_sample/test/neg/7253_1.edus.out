Sentence #1 (11 tokens):
And I am afraid that I cannot imagine why.
[Text=And CharacterOffsetBegin=4 CharacterOffsetEnd=7 PartOfSpeech=CC] [Text=I CharacterOffsetBegin=8 CharacterOffsetEnd=9 PartOfSpeech=PRP] [Text=am CharacterOffsetBegin=10 CharacterOffsetEnd=12 PartOfSpeech=VBP] [Text=afraid CharacterOffsetBegin=13 CharacterOffsetEnd=19 PartOfSpeech=JJ] [Text=that CharacterOffsetBegin=20 CharacterOffsetEnd=24 PartOfSpeech=IN] [Text=I CharacterOffsetBegin=25 CharacterOffsetEnd=26 PartOfSpeech=PRP] [Text=can CharacterOffsetBegin=27 CharacterOffsetEnd=30 PartOfSpeech=MD] [Text=not CharacterOffsetBegin=30 CharacterOffsetEnd=33 PartOfSpeech=RB] [Text=imagine CharacterOffsetBegin=34 CharacterOffsetEnd=41 PartOfSpeech=VB] [Text=why CharacterOffsetBegin=42 CharacterOffsetEnd=45 PartOfSpeech=WRB] [Text=. CharacterOffsetBegin=45 CharacterOffsetEnd=46 PartOfSpeech=.] 
root(ROOT-0, afraid-4)
cc(afraid-4, And-1)
nsubj(afraid-4, I-2)
cop(afraid-4, am-3)
mark(imagine-9, that-5)
nsubj(imagine-9, I-6)
aux(imagine-9, can-7)
neg(imagine-9, not-8)
ccomp(afraid-4, imagine-9)
dep(imagine-9, why-10)
punct(afraid-4, .-11)

Sentence #2 (11 tokens):
It really is a genuinely dire and exceptionally boring film.
[Text=It CharacterOffsetBegin=51 CharacterOffsetEnd=53 PartOfSpeech=PRP] [Text=really CharacterOffsetBegin=54 CharacterOffsetEnd=60 PartOfSpeech=RB] [Text=is CharacterOffsetBegin=61 CharacterOffsetEnd=63 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=64 CharacterOffsetEnd=65 PartOfSpeech=DT] [Text=genuinely CharacterOffsetBegin=66 CharacterOffsetEnd=75 PartOfSpeech=RB] [Text=dire CharacterOffsetBegin=76 CharacterOffsetEnd=80 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=81 CharacterOffsetEnd=84 PartOfSpeech=CC] [Text=exceptionally CharacterOffsetBegin=85 CharacterOffsetEnd=98 PartOfSpeech=RB] [Text=boring CharacterOffsetBegin=99 CharacterOffsetEnd=105 PartOfSpeech=JJ] [Text=film CharacterOffsetBegin=106 CharacterOffsetEnd=110 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=110 CharacterOffsetEnd=111 PartOfSpeech=.] 
root(ROOT-0, is-3)
nsubj(is-3, It-1)
advmod(is-3, really-2)
nsubj(is-3, a-4)
advmod(dire-6, genuinely-5)
amod(a-4, dire-6)
cc(dire-6, and-7)
nn(film-10, exceptionally-8)
amod(film-10, boring-9)
conj(dire-6, film-10)
punct(is-3, .-11)

Sentence #3 (29 tokens):
In some ways it is reminiscent of early science fiction when every set had been knocked up on a Hollywood back lot out of whatever was lying around.
[Text=In CharacterOffsetBegin=116 CharacterOffsetEnd=118 PartOfSpeech=IN] [Text=some CharacterOffsetBegin=119 CharacterOffsetEnd=123 PartOfSpeech=DT] [Text=ways CharacterOffsetBegin=124 CharacterOffsetEnd=128 PartOfSpeech=NNS] [Text=it CharacterOffsetBegin=129 CharacterOffsetEnd=131 PartOfSpeech=PRP] [Text=is CharacterOffsetBegin=132 CharacterOffsetEnd=134 PartOfSpeech=VBZ] [Text=reminiscent CharacterOffsetBegin=135 CharacterOffsetEnd=146 PartOfSpeech=JJ] [Text=of CharacterOffsetBegin=147 CharacterOffsetEnd=149 PartOfSpeech=IN] [Text=early CharacterOffsetBegin=150 CharacterOffsetEnd=155 PartOfSpeech=JJ] [Text=science CharacterOffsetBegin=156 CharacterOffsetEnd=163 PartOfSpeech=NN] [Text=fiction CharacterOffsetBegin=164 CharacterOffsetEnd=171 PartOfSpeech=NN] [Text=when CharacterOffsetBegin=172 CharacterOffsetEnd=176 PartOfSpeech=WRB] [Text=every CharacterOffsetBegin=177 CharacterOffsetEnd=182 PartOfSpeech=DT] [Text=set CharacterOffsetBegin=183 CharacterOffsetEnd=186 PartOfSpeech=NN] [Text=had CharacterOffsetBegin=187 CharacterOffsetEnd=190 PartOfSpeech=VBD] [Text=been CharacterOffsetBegin=191 CharacterOffsetEnd=195 PartOfSpeech=VBN] [Text=knocked CharacterOffsetBegin=196 CharacterOffsetEnd=203 PartOfSpeech=VBN] [Text=up CharacterOffsetBegin=204 CharacterOffsetEnd=206 PartOfSpeech=RP] [Text=on CharacterOffsetBegin=207 CharacterOffsetEnd=209 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=210 CharacterOffsetEnd=211 PartOfSpeech=DT] [Text=Hollywood CharacterOffsetBegin=212 CharacterOffsetEnd=221 PartOfSpeech=NNP] [Text=back CharacterOffsetBegin=222 CharacterOffsetEnd=226 PartOfSpeech=RB] [Text=lot CharacterOffsetBegin=227 CharacterOffsetEnd=230 PartOfSpeech=RB] [Text=out CharacterOffsetBegin=231 CharacterOffsetEnd=234 PartOfSpeech=IN] [Text=of CharacterOffsetBegin=235 CharacterOffsetEnd=237 PartOfSpeech=IN] [Text=whatever CharacterOffsetBegin=238 CharacterOffsetEnd=246 PartOfSpeech=WDT] [Text=was CharacterOffsetBegin=247 CharacterOffsetEnd=250 PartOfSpeech=VBD] [Text=lying CharacterOffsetBegin=251 CharacterOffsetEnd=256 PartOfSpeech=VBG] [Text=around CharacterOffsetBegin=257 CharacterOffsetEnd=263 PartOfSpeech=IN] [Text=. CharacterOffsetBegin=263 CharacterOffsetEnd=264 PartOfSpeech=.] 
root(ROOT-0, reminiscent-6)
prep(reminiscent-6, In-1)
det(ways-3, some-2)
pobj(In-1, ways-3)
nsubj(reminiscent-6, it-4)
cop(reminiscent-6, is-5)
prep(reminiscent-6, of-7)
amod(fiction-10, early-8)
nn(fiction-10, science-9)
pobj(of-7, fiction-10)
advmod(knocked-16, when-11)
det(set-13, every-12)
nsubjpass(knocked-16, set-13)
aux(knocked-16, had-14)
auxpass(knocked-16, been-15)
advcl(reminiscent-6, knocked-16)
prt(knocked-16, up-17)
prep(knocked-16, on-18)
det(Hollywood-20, a-19)
pobj(on-18, Hollywood-20)
advmod(lot-22, back-21)
advmod(knocked-16, lot-22)
prep(lot-22, out-23)
pcomp(out-23, of-24)
nsubj(lying-27, whatever-25)
aux(lying-27, was-26)
pcomp(of-24, lying-27)
advmod(lying-27, around-28)
punct(reminiscent-6, .-29)

Sentence #4 (97 tokens):
From the minuscule and unconvincing set (snipers seem to be about ten meters away) apparently made of plaster, to the actors who are also apparently made of plaster with "amusing" stereotypes painted thinly on top, to the oddly warm pool in a frozen cave, to the survival of the cast uninjured when medium artillery shells burst a few meters away on open ground, and finally the awful script that reads like a training manual more than a film....
    I really cannot say how dull this is.
[Text=From CharacterOffsetBegin=269 CharacterOffsetEnd=273 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=274 CharacterOffsetEnd=277 PartOfSpeech=DT] [Text=minuscule CharacterOffsetBegin=278 CharacterOffsetEnd=287 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=288 CharacterOffsetEnd=291 PartOfSpeech=CC] [Text=unconvincing CharacterOffsetBegin=292 CharacterOffsetEnd=304 PartOfSpeech=JJ] [Text=set CharacterOffsetBegin=305 CharacterOffsetEnd=308 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=309 CharacterOffsetEnd=310 PartOfSpeech=-LRB-] [Text=snipers CharacterOffsetBegin=310 CharacterOffsetEnd=317 PartOfSpeech=NNS] [Text=seem CharacterOffsetBegin=318 CharacterOffsetEnd=322 PartOfSpeech=VBP] [Text=to CharacterOffsetBegin=323 CharacterOffsetEnd=325 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=326 CharacterOffsetEnd=328 PartOfSpeech=VB] [Text=about CharacterOffsetBegin=329 CharacterOffsetEnd=334 PartOfSpeech=IN] [Text=ten CharacterOffsetBegin=335 CharacterOffsetEnd=338 PartOfSpeech=CD] [Text=meters CharacterOffsetBegin=339 CharacterOffsetEnd=345 PartOfSpeech=NNS] [Text=away CharacterOffsetBegin=346 CharacterOffsetEnd=350 PartOfSpeech=RB] [Text=-RRB- CharacterOffsetBegin=350 CharacterOffsetEnd=351 PartOfSpeech=-RRB-] [Text=apparently CharacterOffsetBegin=352 CharacterOffsetEnd=362 PartOfSpeech=RB] [Text=made CharacterOffsetBegin=363 CharacterOffsetEnd=367 PartOfSpeech=VBN] [Text=of CharacterOffsetBegin=368 CharacterOffsetEnd=370 PartOfSpeech=IN] [Text=plaster CharacterOffsetBegin=371 CharacterOffsetEnd=378 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=378 CharacterOffsetEnd=379 PartOfSpeech=,] [Text=to CharacterOffsetBegin=380 CharacterOffsetEnd=382 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=383 CharacterOffsetEnd=386 PartOfSpeech=DT] [Text=actors CharacterOffsetBegin=387 CharacterOffsetEnd=393 PartOfSpeech=NNS] [Text=who CharacterOffsetBegin=394 CharacterOffsetEnd=397 PartOfSpeech=WP] [Text=are CharacterOffsetBegin=398 CharacterOffsetEnd=401 PartOfSpeech=VBP] [Text=also CharacterOffsetBegin=402 CharacterOffsetEnd=406 PartOfSpeech=RB] [Text=apparently CharacterOffsetBegin=407 CharacterOffsetEnd=417 PartOfSpeech=RB] [Text=made CharacterOffsetBegin=418 CharacterOffsetEnd=422 PartOfSpeech=VBN] [Text=of CharacterOffsetBegin=423 CharacterOffsetEnd=425 PartOfSpeech=IN] [Text=plaster CharacterOffsetBegin=426 CharacterOffsetEnd=433 PartOfSpeech=NN] [Text=with CharacterOffsetBegin=434 CharacterOffsetEnd=438 PartOfSpeech=IN] [Text=`` CharacterOffsetBegin=439 CharacterOffsetEnd=440 PartOfSpeech=``] [Text=amusing CharacterOffsetBegin=440 CharacterOffsetEnd=447 PartOfSpeech=JJ] [Text='' CharacterOffsetBegin=447 CharacterOffsetEnd=448 PartOfSpeech=''] [Text=stereotypes CharacterOffsetBegin=449 CharacterOffsetEnd=460 PartOfSpeech=NNS] [Text=painted CharacterOffsetBegin=461 CharacterOffsetEnd=468 PartOfSpeech=VBD] [Text=thinly CharacterOffsetBegin=469 CharacterOffsetEnd=475 PartOfSpeech=RB] [Text=on CharacterOffsetBegin=476 CharacterOffsetEnd=478 PartOfSpeech=IN] [Text=top CharacterOffsetBegin=479 CharacterOffsetEnd=482 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=482 CharacterOffsetEnd=483 PartOfSpeech=,] [Text=to CharacterOffsetBegin=484 CharacterOffsetEnd=486 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=487 CharacterOffsetEnd=490 PartOfSpeech=DT] [Text=oddly CharacterOffsetBegin=491 CharacterOffsetEnd=496 PartOfSpeech=RB] [Text=warm CharacterOffsetBegin=497 CharacterOffsetEnd=501 PartOfSpeech=JJ] [Text=pool CharacterOffsetBegin=502 CharacterOffsetEnd=506 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=507 CharacterOffsetEnd=509 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=510 CharacterOffsetEnd=511 PartOfSpeech=DT] [Text=frozen CharacterOffsetBegin=512 CharacterOffsetEnd=518 PartOfSpeech=JJ] [Text=cave CharacterOffsetBegin=519 CharacterOffsetEnd=523 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=523 CharacterOffsetEnd=524 PartOfSpeech=,] [Text=to CharacterOffsetBegin=525 CharacterOffsetEnd=527 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=528 CharacterOffsetEnd=531 PartOfSpeech=DT] [Text=survival CharacterOffsetBegin=532 CharacterOffsetEnd=540 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=541 CharacterOffsetEnd=543 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=544 CharacterOffsetEnd=547 PartOfSpeech=DT] [Text=cast CharacterOffsetBegin=548 CharacterOffsetEnd=552 PartOfSpeech=NN] [Text=uninjured CharacterOffsetBegin=553 CharacterOffsetEnd=562 PartOfSpeech=JJ] [Text=when CharacterOffsetBegin=563 CharacterOffsetEnd=567 PartOfSpeech=WRB] [Text=medium CharacterOffsetBegin=568 CharacterOffsetEnd=574 PartOfSpeech=NN] [Text=artillery CharacterOffsetBegin=575 CharacterOffsetEnd=584 PartOfSpeech=NN] [Text=shells CharacterOffsetBegin=585 CharacterOffsetEnd=591 PartOfSpeech=NNS] [Text=burst CharacterOffsetBegin=592 CharacterOffsetEnd=597 PartOfSpeech=VBP] [Text=a CharacterOffsetBegin=598 CharacterOffsetEnd=599 PartOfSpeech=DT] [Text=few CharacterOffsetBegin=600 CharacterOffsetEnd=603 PartOfSpeech=JJ] [Text=meters CharacterOffsetBegin=604 CharacterOffsetEnd=610 PartOfSpeech=NNS] [Text=away CharacterOffsetBegin=611 CharacterOffsetEnd=615 PartOfSpeech=RB] [Text=on CharacterOffsetBegin=616 CharacterOffsetEnd=618 PartOfSpeech=IN] [Text=open CharacterOffsetBegin=619 CharacterOffsetEnd=623 PartOfSpeech=JJ] [Text=ground CharacterOffsetBegin=624 CharacterOffsetEnd=630 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=630 CharacterOffsetEnd=631 PartOfSpeech=,] [Text=and CharacterOffsetBegin=632 CharacterOffsetEnd=635 PartOfSpeech=CC] [Text=finally CharacterOffsetBegin=636 CharacterOffsetEnd=643 PartOfSpeech=RB] [Text=the CharacterOffsetBegin=644 CharacterOffsetEnd=647 PartOfSpeech=DT] [Text=awful CharacterOffsetBegin=648 CharacterOffsetEnd=653 PartOfSpeech=JJ] [Text=script CharacterOffsetBegin=654 CharacterOffsetEnd=660 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=661 CharacterOffsetEnd=665 PartOfSpeech=WDT] [Text=reads CharacterOffsetBegin=666 CharacterOffsetEnd=671 PartOfSpeech=VBZ] [Text=like CharacterOffsetBegin=672 CharacterOffsetEnd=676 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=677 CharacterOffsetEnd=678 PartOfSpeech=DT] [Text=training CharacterOffsetBegin=679 CharacterOffsetEnd=687 PartOfSpeech=NN] [Text=manual CharacterOffsetBegin=688 CharacterOffsetEnd=694 PartOfSpeech=NN] [Text=more CharacterOffsetBegin=695 CharacterOffsetEnd=699 PartOfSpeech=JJR] [Text=than CharacterOffsetBegin=700 CharacterOffsetEnd=704 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=705 CharacterOffsetEnd=706 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=707 CharacterOffsetEnd=711 PartOfSpeech=NN] [Text=... CharacterOffsetBegin=711 CharacterOffsetEnd=715 PartOfSpeech=:] [Text=I CharacterOffsetBegin=720 CharacterOffsetEnd=721 PartOfSpeech=PRP] [Text=really CharacterOffsetBegin=722 CharacterOffsetEnd=728 PartOfSpeech=RB] [Text=can CharacterOffsetBegin=729 CharacterOffsetEnd=732 PartOfSpeech=MD] [Text=not CharacterOffsetBegin=732 CharacterOffsetEnd=735 PartOfSpeech=RB] [Text=say CharacterOffsetBegin=736 CharacterOffsetEnd=739 PartOfSpeech=VB] [Text=how CharacterOffsetBegin=740 CharacterOffsetEnd=743 PartOfSpeech=WRB] [Text=dull CharacterOffsetBegin=744 CharacterOffsetEnd=748 PartOfSpeech=JJ] [Text=this CharacterOffsetBegin=749 CharacterOffsetEnd=753 PartOfSpeech=DT] [Text=is CharacterOffsetBegin=754 CharacterOffsetEnd=756 PartOfSpeech=VBZ] [Text=. CharacterOffsetBegin=756 CharacterOffsetEnd=757 PartOfSpeech=.] 
root(ROOT-0, say-92)
prep(say-92, From-1)
det(set-6, the-2)
amod(set-6, minuscule-3)
cc(minuscule-3, and-4)
conj(minuscule-3, unconvincing-5)
pobj(From-1, set-6)
punct(snipers-8, -LRB--7)
nsubj(seem-9, snipers-8)
rcmod(set-6, seem-9)
aux(meters-14, to-10)
cop(meters-14, be-11)
quantmod(ten-13, about-12)
num(meters-14, ten-13)
xcomp(seem-9, meters-14)
advmod(meters-14, away-15)
punct(meters-14, -RRB--16)
advmod(made-18, apparently-17)
partmod(meters-14, made-18)
prep(made-18, of-19)
pobj(of-19, plaster-20)
punct(meters-14, ,-21)
prep(meters-14, to-22)
det(actors-24, the-23)
pobj(to-22, actors-24)
nsubjpass(made-29, who-25)
auxpass(made-29, are-26)
advmod(made-29, also-27)
advmod(made-29, apparently-28)
rcmod(actors-24, made-29)
prep(made-29, of-30)
pobj(of-30, plaster-31)
prep(made-29, with-32)
punct(stereotypes-36, ``-33)
amod(stereotypes-36, amusing-34)
punct(stereotypes-36, ''-35)
pobj(with-32, stereotypes-36)
partmod(stereotypes-36, painted-37)
advmod(painted-37, thinly-38)
prep(thinly-38, on-39)
pobj(on-39, top-40)
punct(painted-37, ,-41)
prep(painted-37, to-42)
det(pool-46, the-43)
amod(pool-46, oddly-44)
amod(pool-46, warm-45)
pobj(to-42, pool-46)
prep(painted-37, in-47)
det(cave-50, a-48)
amod(cave-50, frozen-49)
pobj(in-47, cave-50)
punct(stereotypes-36, ,-51)
prep(stereotypes-36, to-52)
det(survival-54, the-53)
pobj(to-52, survival-54)
prep(survival-54, of-55)
det(cast-57, the-56)
pobj(of-55, cast-57)
amod(cast-57, uninjured-58)
advmod(burst-63, when-59)
amod(shells-62, medium-60)
nn(shells-62, artillery-61)
nsubj(burst-63, shells-62)
advcl(uninjured-58, burst-63)
det(meters-66, a-64)
amod(meters-66, few-65)
dobj(burst-63, meters-66)
advmod(burst-63, away-67)
prep(burst-63, on-68)
amod(ground-70, open-69)
pobj(on-68, ground-70)
punct(stereotypes-36, ,-71)
cc(stereotypes-36, and-72)
advmod(stereotypes-36, finally-73)
det(script-76, the-74)
amod(script-76, awful-75)
dep(stereotypes-36, script-76)
nsubj(reads-78, that-77)
rcmod(script-76, reads-78)
prep(reads-78, like-79)
det(manual-82, a-80)
nn(manual-82, training-81)
pobj(like-79, manual-82)
amod(manual-82, more-83)
prep(more-83, than-84)
det(film-86, a-85)
pobj(than-84, film-86)
punct(say-92, ...-87)
nsubj(say-92, I-88)
advmod(say-92, really-89)
aux(say-92, can-90)
neg(say-92, not-91)
advmod(dull-94, how-93)
dep(is-96, dull-94)
nsubj(is-96, this-95)
ccomp(say-92, is-96)
punct(say-92, .-97)

Sentence #5 (24 tokens):
Even the opportunity to see whether the young James Dean survived wasn't enough to keep me watching for more than an hour.
[Text=Even CharacterOffsetBegin=762 CharacterOffsetEnd=766 PartOfSpeech=RB] [Text=the CharacterOffsetBegin=767 CharacterOffsetEnd=770 PartOfSpeech=DT] [Text=opportunity CharacterOffsetBegin=771 CharacterOffsetEnd=782 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=783 CharacterOffsetEnd=785 PartOfSpeech=TO] [Text=see CharacterOffsetBegin=786 CharacterOffsetEnd=789 PartOfSpeech=VB] [Text=whether CharacterOffsetBegin=790 CharacterOffsetEnd=797 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=798 CharacterOffsetEnd=801 PartOfSpeech=DT] [Text=young CharacterOffsetBegin=802 CharacterOffsetEnd=807 PartOfSpeech=JJ] [Text=James CharacterOffsetBegin=808 CharacterOffsetEnd=813 PartOfSpeech=NNP] [Text=Dean CharacterOffsetBegin=814 CharacterOffsetEnd=818 PartOfSpeech=NNP] [Text=survived CharacterOffsetBegin=819 CharacterOffsetEnd=827 PartOfSpeech=VBD] [Text=was CharacterOffsetBegin=828 CharacterOffsetEnd=831 PartOfSpeech=VBD] [Text=n't CharacterOffsetBegin=831 CharacterOffsetEnd=834 PartOfSpeech=RB] [Text=enough CharacterOffsetBegin=835 CharacterOffsetEnd=841 PartOfSpeech=RB] [Text=to CharacterOffsetBegin=842 CharacterOffsetEnd=844 PartOfSpeech=TO] [Text=keep CharacterOffsetBegin=845 CharacterOffsetEnd=849 PartOfSpeech=VB] [Text=me CharacterOffsetBegin=850 CharacterOffsetEnd=852 PartOfSpeech=PRP] [Text=watching CharacterOffsetBegin=853 CharacterOffsetEnd=861 PartOfSpeech=VBG] [Text=for CharacterOffsetBegin=862 CharacterOffsetEnd=865 PartOfSpeech=IN] [Text=more CharacterOffsetBegin=866 CharacterOffsetEnd=870 PartOfSpeech=JJR] [Text=than CharacterOffsetBegin=871 CharacterOffsetEnd=875 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=876 CharacterOffsetEnd=878 PartOfSpeech=DT] [Text=hour CharacterOffsetBegin=879 CharacterOffsetEnd=883 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=883 CharacterOffsetEnd=884 PartOfSpeech=.] 
root(ROOT-0, enough-14)
advmod(opportunity-3, Even-1)
det(opportunity-3, the-2)
nsubj(enough-14, opportunity-3)
aux(see-5, to-4)
infmod(opportunity-3, see-5)
mark(survived-11, whether-6)
det(Dean-10, the-7)
amod(Dean-10, young-8)
nn(Dean-10, James-9)
nsubj(survived-11, Dean-10)
ccomp(see-5, survived-11)
cop(enough-14, was-12)
neg(enough-14, n't-13)
aux(keep-16, to-15)
xcomp(enough-14, keep-16)
nsubj(watching-18, me-17)
xcomp(keep-16, watching-18)
prep(watching-18, for-19)
pobj(for-19, more-20)
prep(more-20, than-21)
det(hour-23, an-22)
pobj(than-21, hour-23)
punct(enough-14, .-24)

Sentence #6 (11 tokens):
This really is one to be avoided at ALL costs.
[Text=This CharacterOffsetBegin=889 CharacterOffsetEnd=893 PartOfSpeech=DT] [Text=really CharacterOffsetBegin=894 CharacterOffsetEnd=900 PartOfSpeech=RB] [Text=is CharacterOffsetBegin=901 CharacterOffsetEnd=903 PartOfSpeech=VBZ] [Text=one CharacterOffsetBegin=904 CharacterOffsetEnd=907 PartOfSpeech=CD] [Text=to CharacterOffsetBegin=908 CharacterOffsetEnd=910 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=911 CharacterOffsetEnd=913 PartOfSpeech=VB] [Text=avoided CharacterOffsetBegin=914 CharacterOffsetEnd=921 PartOfSpeech=VBN] [Text=at CharacterOffsetBegin=922 CharacterOffsetEnd=924 PartOfSpeech=IN] [Text=ALL CharacterOffsetBegin=925 CharacterOffsetEnd=928 PartOfSpeech=NN] [Text=costs CharacterOffsetBegin=929 CharacterOffsetEnd=934 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=934 CharacterOffsetEnd=935 PartOfSpeech=.] 
root(ROOT-0, one-4)
nsubj(one-4, This-1)
advmod(one-4, really-2)
cop(one-4, is-3)
aux(avoided-7, to-5)
auxpass(avoided-7, be-6)
infmod(one-4, avoided-7)
prep(avoided-7, at-8)
nn(costs-10, ALL-9)
pobj(at-8, costs-10)
punct(one-4, .-11)

